pcdsdevices/spectrometer.py

```diff
@@ -5,7 +5,7 @@
 from ophyd.device import FormattedComponent as FCpt
 
 from .device import GroupDevice
-from .epics_motor import BeckhoffAxisNoOffset, IMS
+from .epics_motor import BeckhoffAxisNoOffset, IMS, BeckhoffAxis
```

Comment:
`isort` should have sorted these imports when the file was committed.
Did you do `pre-commit install` in your cloned pcdsdevices repository?

I won't let this hold up the PR so long as the tests pass, but it's good practice to use it in the future.
For reference:
https://confluence.slac.stanford.edu/display/PCDS/How+to+add+a+device+to+pcdsdevices#Howtoaddadevicetopcdsdevices-Pre-commit

Reply:
No I've never done pre-commit install, first time I've heard of this. Thanks for the heads up Ken!